src/components/controls/keyboard-controls.js

```diff
@@ -0,0 +1,94 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
Yeah, I get what you mean. I really want to commit to this Tough Choice though.

The future I imagine, I don't think VR can bring along flat experiences; it's a different medium. If a developer wants their experience to be accessible from a non-VR experience, then they can put in and extra effort and design it to also work on flat. But for most VR experiences that are designed only for VR (room-scale, Touch/Vive controllers), which is what A-Frame is focused on, non-VR users will not be able to experience them in any meaningful way. 

Flat experiences, almost like the current form of Cardboard, are sort of distractions for true WebVR. Defaults are important is almost a case that we *should* nuke default flat experiences. Because with WASD controls available, most developers are content with that form of locomotion and don't optimize the UX for VR. I think the middle ground is having it available in a separate repo if a developer wants it.

If we go this route, we'd probably just nuke default WASD controls for camera as well, and then really prototype better locomotion. I think this would possibly hurt A-Frame's popularity since a lot of people are using it for just flat experiences, but it's better for the long-term. Flat web games never really took off anyways, so might be a good idea to drop that baggage.